app/src/main/resources/eip-client.properties

```diff
@@ -6,6 +6,15 @@ spring.banner.location=classpath:banner.txt
 spring.jpa.open-in-view=false
 # ----------------------------------------------------------------------------------------------------------------------
 
+## --- Monitoring Configuration ------------------------------------------------------------------------------------------
+management.endpoints.web.exposure.include=*
+management.endpoints.web.exposure.include=prometheus,health,info,metric
```

Comment:
Do we want to expose all endpoints? if that's the case line 11 is redundant and should be removed as the previous line already exposes all endpoints. If not don't expose all endpoints(remove line 10).